src/components/PageBlocks/RecordListConfigurator.vue

```diff
@@ -31,15 +31,8 @@
         <field-picker
           :module="recordListModule"
           :fields.sync="options.fields"
-          :labels="{
-            searchPlaceholder: $t('field:selector.search'),
-            availableFields: $t('field:selector.available'),
-            selectAllFields: $t('field:selector.selectAll'),
-            selectedFields: $t('field:selector.selected'),
-            unselectAllFields: $t('field:selector.unselectAll'),
-            systemField: $t('field:selector.systemField'),
-          }"
-          max-height="max-height: 30vh;"
+          style="max-height: 40vh;"
```

Comment:
A minor thing, style should be after class. No need to change but for the future.
We'll do a cleanup of repos soon where all eslinters are synced. This rule is already on some repos